src/tests/Loader/classloader/explicitlayout/Regressions/empty/explicitStruct_empty.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[StructLayout(LayoutKind.Explicit)]
+public struct S
```

Comment:
Just for my education - does the same logic hold for zero-sized classes with explicit layout i.o.w. should we be also testing correct behavior in the presence of inheritance?

Reply:
I believe zero-sized classes are already bumped up to at least 1-byte elsewhere, but to make sure I've added some more test cases to make sure.